_summaries/ulyanov2016instance_1.md

```diff
@@ -1,19 +1,64 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: ulyanov2016instance
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: rll2396
+score: 4
 ---
 
 TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+* Core Idea
+
+    Instance normalization can help a feed-forward generator network achieve better performance at stylizing images.
+
+* Context
+    * Gatys et al. (2016) showed promising results for using CNNs for the task of image stylization
+
+    * The authors (2016) previously showed that the same task can be achieved hundreds of times faster at a cost to the quality of the images generated
+
+    * In this paper they show that changing the normalization from batch to instance can improve the quality
```

Comment:
For context is there a reason the author has decided to change the normalization? is it because the computational efficiency ? 
I think adding some pointers to the history of the problem will make a case for the good results listed here.